doc/sphinx/source/recipes/recipe_ref.rst

```diff
@@ -82,3 +88,11 @@ Example plots:
 
    Zonal mean vertically resolved ozone climatology from CNRM-ESM2-1 compared with ESACCI-OZONE
    for the years 1990 to 2000. Created with recipe_ref_ozone.yml.
+
+.. _fig_ref_7:
+.. figure::  /recipes/figures/ref/timeseries_tas_ambiguous_dataset_Amon_historical_r1i1p1f1.png
+   :align:   center
+   :width:   8cm
+
+   Time series of near-surface air temperature anomalies from MIROC6 compared with HadCRUT5
+   for N.Europe for the years 1980 to 2014 (refernce period 1980 to 2009). Created with recipe_model_benchmarking_timeseries_region.yml.
```

Comment:
There is a typo in the word "reference".

Reply:
Thanks, I removed it